go.mod

```diff
@@ -10,11 +10,8 @@ require (
 	github.com/dapr/components-contrib v0.0.0-20191128051247-c4bd2a47eae0
 	github.com/eapache/go-resiliency v1.2.0 // indirect
 	github.com/emicklei/go-restful v2.10.0+incompatible // indirect
-	github.com/evanphx/json-patch v4.2.0+incompatible // indirect
 	github.com/frankban/quicktest v1.5.0 // indirect
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-openapi/jsonreference v0.19.3 // indirect
-	github.com/go-openapi/spec v0.19.3 // indirect
```

Comment:
Any reason to remove these?

Reply:
I did not explicitly remove this . Probably I need to rebase.

Reply:
rebased now.